torch/_inductor/fx_passes/split_cat.py

```diff
@@ -602,6 +602,7 @@ def simplify(
             graph, split_node, next_users, user_inputs_list_new, transform_params_list  # type: ignore[arg-type]
         )
         self.erase_old_nodes(graph, split_node, next_users)  # type: ignore[arg-type]
+        counters["inductor"]["unbind_stack_pass"] += 1
```

Comment:
is KeyError possible here?  https://fburl.com/code/pgboncnv

Reply:
Good question. The counters is defined here https://fburl.com/code/mwm7l9dm. Will not have KeyError problem.